### PR TITLE
Awesome books: with ES6

### PR DESCRIPTION
**Pull request by RitaDee**
### In this project, the following was implemented👇🏾👇🏾

- Used ES6 syntax.
- Divided my code into modules. Saved each module in a separate file in the modules dir. Imported modules in the index.js file. 
- Kept all functionalities of the app without errors.
- Refactor my methods and functions to arrow functions.
- Used a library [Luxon](https://moment.github.io/luxon/#/install?id=es6) for current time and date.

### CODE REVIEW

File: modules/books.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  addBook(Booktitle, Bookauthor) {
+    if (Booktitle && Bookauthor) {
+      const newBook = { title: Booktitle, author: Bookauthor };
+      this.collection.push(newBook);
+    }
+  }
+
+  removeBook(index) {
+    this.collection.splice(index, 1);
+  }
```
Comment (ha-manel) [inline modules/books.js]: - [ ] Kindly use `arrow functions` for all the functions in the project to follow ES6 syntax as required 👍 

```
addBook = (bookTitle, bookAuthor) => {
}
```

File: modules/show.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const addSection = document.getElementById('add-book');
+const contactBtn = document.querySelector('#contact-link');
+const contactSection = document.querySelector('.contact-section');
+
+// function showDateTime() {
+//   const dateNew = new Date();
+//   date.innerHTML = dateNew.toLocaleString('en-GB');
+// }
+// showDateTime();
+// setInterval(showDateTime, 1);
```
Comment (ha-manel) [inline modules/show.js]: - [ ] Kindly remove commented unused pieces of code from the project for a cleaner, more professional code 💯

**Review by ha-manel — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES ♻️

## Hi @RitaDee 👋 
Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍  



## Highlights:  
* All tests passed ✔️
* Meaningful commit messages ✔️ 
* Professional README file ✔️
* The app is working nicely 💯 

## Required Changes:
_Kindly check the comments below_

------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mukhammadrizooff — APPROVED** [review]
Hi @RitaDee, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
- No linter errors ✔️ 
- Good implementations ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**